### ISSUE TITLE
Non-root policy softmax temperature

### ISSUE THREAD

**Issue by shindavid** [shindavid/AlphaZeroArcade#55]
Examining the source code, it appears that KataGo applies softmax-temperature to the raw neural network output for non-root nodes, not just for the root node.

This should be experimented with.
